src/util/storage/register_storage.hpp

```diff
@@ -67,6 +73,11 @@ namespace cyphal::support
     }
     return std::nullopt;
 }
+[[nodiscard]] inline std::optional<platform::storage::Error> load(const platform::storage::interface::KeyValueStorage& kv,
+                                                                  registry::IIntrospectableRegistry&  rgy)
+{
+  return load(kv, rgy, nullptr);
+}
```

Comment:
It almost looks like you want to use a default argument in the first definition above.

Reply:
I don't like default arguments much because they are a hindrance when you want to extend the parameter list in the future.